.babelrc

```diff
@@ -1,3 +1,4 @@
 {
-  "presets": ["env", "react"]
+  "presets": ["env", "react"],
+  "plugins": ["transform-object-rest-spread"]
```

Comment:
not going to request to remove this cause you still have an error, it's a pain to download and setup again.
which hopefully you guys would fix tomorrow and after it ( bug fixing days )